packages/utils/src/cmds/transform_translate.js

```diff
@@ -0,0 +1,381 @@
+                )
+                log.debug(
+                    `If you want to override the existing files, use the "--override-existing-files" option`
+                )
+
+                continue
+            } else if (!fs.existsSync(newLanguageFilePath)) {
+                newContents +=
+                    language === argv.primaryLanguage
+                        ? getTemplateMainLanguage()
+                        : getTemplateAlternativeLanguage(argv.appName, language)
+            }
+
+            for (const key in languageTranslations) {
+                if (languageTranslations.hasOwnProperty(key)) {
+                    if (!translations[argv.primaryLanguage][key]) {
+                        argv.logMissingKeys &&
+                            log.debug(
+                                `Original translation missing for key "${key}" of language "${language}"`
+                            )
+                        continue
+                    }
+
+                    const originalTranslation =
+                        translations[argv.primaryLanguage][key]
+                    const translation =
+                        language === argv.primaryLanguage
+                            ? ''
+                            : languageTranslations[key]
+
+                    newContents += '\n\n'
+                    if (originalTranslation.length < 77) {
+                        newContents += `msgid "${originalTranslation}"\n`
+                    } else {
+                        newContents += `msgid ""\n`
+                        const newLines = splitTranslation(
+                            '',
+                            originalTranslation
+                        )
+                        newLines.forEach(translationPart => {
+                            newContents += `"${unescape(translationPart)}"\n`
+                        })
+                    }
+
+                    if (translation.length < 77) {
+                        newContents += `msgstr "${translation}"`
+                    } else {
+                        newContents += `msgstr ""\n`
+                        const newLines = splitTranslation(
+                            originalTranslation,
+                            translation
+                        )
+                        newLines.forEach(translationPart => {
+                            newContents += `"${unescape(translationPart)}"\n`
+                        })
+                    }
+                }
+            }
+
+            argv.verbose &&
+                log.debug(
+                    `${
+                        shouldAppendContents ? 'Appending' : 'Writing'
+                    } to "${newLanguageFilePath}"`
+                )
+            fs.writeFileSync(newLanguageFilePath, newContents, {
+                flag: shouldAppendContents ? 'a' : 'w',
+            })
+        }
+    }
+
+    argv.verbose && log.debug('Creating translation files :: Done')
```

Comment:
`reporter.debug` already handles only showing debug logs when `--verbose` is passed, so you don't need to use `argv.verbose && ...` everywhere